src/components/PopupTrigger.js

```diff
@@ -0,0 +1,152 @@
+import _ from 'lodash/fp'
```

Comment:
this whole file is a copy/paste job from terra-ui, kept as similar as possible without importing or defining unnecessary code